src/main/java/it/pagopa/pn/deliverypush/middleware/externalclient/pnclient/externalchannel/ExternalChannelSendClientImpl.java

```diff
@@ -178,7 +179,7 @@ private void sendNotificationEMAIL(String requestId, NotificationInt notificatio
 
             log.info("[exit] sendNotificationEMAIL address={} requestId={} recipient={}", LogUtils.maskEmailAddress(digitalAddress.getAddress()), requestId, LogUtils.maskGeneric(recipientInt.getDenomination()));
         } catch (Exception e) {
-            throw new PnInternalException("error sending EMAIL notification", e);
+            throw new PnInternalException("error sending EMAIL notification", ERROR_CODE_DELIVERYPUSH_SENDEMAILNOTIFICATIONFAILED);
```

Comment:
- Nei casi in cui l'exception è presente va sempre utilizzato il costruttore di PnInternalException che prende anche il throwable come parametro di ingresso, altrimenti ci si perde l'informazione sull'exception.
- Aggiungere Log errore con exception